.circleci/config.yml

```
@@ -70,6 +93,156 @@ jobs:
           only_for_branches: master
       - store_artifacts:
           path: /tmp/tap-mongodb
+  build_mongo_4_4:
+    executor: tap_tester_mongo_4_4
+    steps:
+      - checkout
+      - run:
+          name: 'Install Dockerize'
+          command: |
+            wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+            && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+            && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.3.0
+      - run:
+          name: 'Wait for Mongo'
+          command: |
+            dockerize -wait tcp://127.0.0.1:27017 -timeout 1m
+            sleep 10
+      - run:
+          name: 'Setup Mongo'
+          command: |
+            aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/tap_tester_sandbox tap-tester.env
+            source tap-tester.env
+            wget -qO - https://www.mongodb.org/static/pgp/server-5.0.asc | apt-key add -
+            echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/5.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+            apt-get update
+            apt-get install -y mongodb-org-shell mongodb-mongosh mongodb-org
+            mongosh -u $TAP_MONGODB_USER -p $TAP_MONGODB_PASSWORD --authenticationDatabase admin --eval "rs.initiate({_id: \"rs0\", members: [{_id: 0, host: \"$TAP_MONGODB_HOST:$TAP_MONGODB_PORT\"}]})"
```

In case we care about line length
```suggestion
            mongosh -u $TAP_MONGODB_USER \
                    -p $TAP_MONGODB_PASSWORD \
                    --authenticationDatabase admin \
                    --eval "rs.initiate({_id: \"rs0\", members: [{_id: 0, host: \"$TAP_MONGODB_HOST:$TAP_MONGODB_PORT\"}]})"
```

---

Split all three of these lines along with the "echo "deb .." | tee /*.log" lines